python/lsst/drp/tasks/metadetection_shear.py

```diff
@@ -117,14 +124,20 @@ class MetadetectionShearConfig(
 ):
     """Configuration definition for MetadetectionShearTask."""
 
+    from lsst.meas.base import SkyMapIdGeneratorConfig
+
     required_bands = ListField[str](
         "Bands expected to be present.  Cells with one or more of these bands "
         "missing will be skipped.  Bands other than those listed here will "
```

Comment:
I don't think this docstring is consistent with what happens. If `g` is specified by the coadd data doesn't exist, I think a `NoWorkFound` gets raised and nothing gets processed. I think that's the correct behavior and the docstring needs to reflect that.

Reply:
Jim wrote that doc string, maybe he can comment

Reply:
I wrote what I expect to happen.  Happy to have it changed to what will happen.